src/cipher/longrequest/tests.py

```diff
@@ -1198,10 +1199,10 @@ def setUp(test=None):
     sys._current_frames.return_value = collections.defaultdict(
         sys._getframe)
     traceback.print_stack = lambda frame, file: file.write(
-        '  File "module.py", line 69, in main\n'
-        '    do_stuff()\n'
-        '  File "submodule.py", line 42, in helper\n'
-        '    endless_loop()\n')
+        b'  File "module.py", line 69, in main\n'
+        b'    do_stuff()\n'
+        b'  File "submodule.py", line 42, in helper\n'
+        b'    endless_loop()\n')
```

Comment:
This is a bit surprising to me -- does `traceback.print_stack()` write _bytes_ on Python 3?  (Or is it just that we're mocking sys.stdout in the doctest with a `BytesIO` instance or something?)

Reply:
Yeah, getThreadTraceback() in longrequest.py writes it to BytesIO, yes, and I don't think it should, I think it should use StringIO under Python3. Good gatch.